### ISSUE TITLE
Why setting 'params.requires_grad=False' for target_network?

### ISSUE THREAD

**Issue by luofan18** [SSL92/hyperIQA#5]
https://github.com/SSL92/hyperIQA/blob/3c4fe1366f1cc851efc64015f293aaf7fc7eaa0c/HyerIQASolver.py#L50-L56

Hi, thanks for releasing your code. I find that you set requires_grad=False for target_networks. This is a little confusing.  Since the parameter of target network are generated by hyperNet, would setting requires_grad to False block loss of targetnet backpropogating to parameters of hyperNet and make the parameters of hypernet do not update?

**Comment by luofan18** [comment]
I just tested model_target.parameters() and turned out it is empty. So, this piece of code will has no effect on final result.
